paddle/phi/kernels/cpu/gaussian_kernel.cc

```diff
@@ -48,7 +48,38 @@ void GaussianKernel(const Context& dev_ctx,
   }
 }
 
+template <typename T, typename Context>
+void GaussianInplaceKernel(const Context& dev_ctx,
+                           const DenseTensor& x,
+                           float mean,
+                           float std,
```

Comment:
if dtype of `x` is double, implicit type conversion will be performed here, resulting in a loss of accuracy.

Reply:
As above